packages/camera/camera/lib/src/camera_controller.dart

```diff
@@ -29,6 +29,10 @@ Future<List<CameraDescription>> availableCameras() async {
   return CameraPlatform.instance.availableCameras();
 }
 
+// TODO(stuartmorgan): Remove this once the package requires 2.10, where the
+// dart:async `unawaited` accepts a nullable future.
+void _unawaited(Future<void>? future) {}
```

Comment:
QQ: What about using `// ignore:unawaited_futures` instead of the `_unawaited/unawaited` function? AFAIK the function only slightly changes the syntax of the code for it to pass analysis... So we could as well be explicit telling the world that we want to ignore the lint where needed (and maybe then we don't have to revisit this later?).

Reply:
I thought about that, but since the SDK has `unawaited` as a preferred way to do this, following that model seemed cleaner and more consistent with other code.

(Technically we could just remove this since we don't have that lint enabled right now, but I have on my TODO list turning it on and seeing how it works out. We took our settings from Flutter which has it disabled with a note about "too many false positives", but I don't know if that would be true for us, and we have caught bugs with it before.)